libvast/src/format/zeek.cpp

```diff
@@ -276,8 +277,18 @@ caf::error reader::read_impl(size_t max_events, size_t max_slice_size,
   size_t produced = 0;
   // Loop until reaching EOF or the configured limit of records.
   while (produced < max_events) {
-    // Advance line range and check for EOF.
-    lines_->next();
+    // Advance line range and check for EOF. (In the first iteration, the
+    // current line is the last line of the header)
+    // If we already have some events, set a read timeout to ensure downstream
+    // sees them even if there are currently no further events in the stream.
+    if (builder_->rows() > 0) {
+      using namespace std::chrono_literals;
+      bool timeout = lines_->next_timeout(500ms); // TODO: Make it configurable.
```

Comment:
Do you plan on resolving this within this PR?

Reply:
I can add it if you think it makes sense, but I also wouldn't feel too bad to leave it hard-coded for now since I can't really think of a reason for a user to change the setting, and it's not strictly related to fixing continous import of zeek logs.

Reply:
Maybe this can just be a constant in the defaults namespace. No need to add an option for it.